tutorials/migrating-rails-app-from-heroku-to-google-appengine.md

```diff
@@ -304,7 +297,7 @@ Configuration.
 If you have any other sensitive configuration fields, copy them over in the
 same way.
 
-Next we'll instruct your application to load this configuration into
+Next we instruct your application to load this configuration into
```

Comment:
I think we should just get rid of "we" altogether.

Reply:
I agree, but as the author, I leave that up to you.

Reply:
Done